docs/pushmsg.md

```diff
@@ -13,13 +13,20 @@ An application can be added via
 To authenticate as an application you need the application token.
 The token is returned in the REST request and is viewable in the WebUI.
 
-Now you can simply use [curl](https://curl.haxx.se/), [HTTPie](https://httpie.org/) or any other http-client to push messages.
+Now you can simply use [curl](https://curl.haxx.se/), [HTTPie](https://httpie.org/) or any other installed http-client to push messages.
 
 ```bash
 $ curl "https://push.example.de/message?token=<apptoken>" -F "title=my title" -F "message=my message" -F "priority=5"
 $ http -f POST "https://push.example.de/message?token=<apptoken>" title="my title" message="my message" priority="5"
 ```
 
+On Microsoft PowerShell, you could alternatively use the built-in `Invoke-RestMethod` or `Invoke-WebRequest` cmdlets.
```

Comment:
Is there a reason for adding it here instead of https://gotify.net/docs/more-pushmsg?

Reply:
My thought was to cover most systems in the main page (and most systems either have curl or powershell), but I'm okay with putting it in the more pages too.

Reply:
:+1:, just wanted to make sure it was a conscious decision. I'm okay with both, your choice (:.